src/main/kotlin/minesweeper/domain/area/Area.kt

```diff
@@ -0,0 +1,9 @@
+package minesweeper.domain.area
+
+data class Area(private val width: Width, private val height: Height) {
+    fun width(): Int = width.width
+
+    fun height(): Int = height.height
```

Comment:
프로퍼티를 사용할까, 메서드를 사용할까 고민했는데  아래 특징을 고려해서 메서드 호출로 선언했습니다. 

* 프로퍼티 : 처음 초기화시에 값이 초기화됨 
* 메서드 : 호출때마다 값을 계산해서 반환 
 
매번 호출하는 메서드가 안좋을 수 있지만,    
참조하고 있는 객체 그리고 객체 내부 필드가 불변이 아닌 가변이라 가정한다면   
바뀐 값을 반영 못한 상태로 getter 를 호출하는 것이므로 메서드 호출로 선언했습니다

Reply:
바뀐 값을 반영하고 싶다면  get() 을 이용하는건 어떨까요?
https://kotlinlang.org/docs/properties.html#declaring-properties
```suggestion
   data class Area(private val _width: Width, private val height: Height) {
    val width get() = _width.width


```

Reply:
get()을 이용한 방식도 고려해봤는데  
이 부분에 대해서 width 를 반환하는 경우도 있지 않을까 해서 우선은 메서드로 작성을 했습니다.  
허나, 현재는 반환하는 경우가 없으므로 get()으로 바꿔서 진행해보겠습니다